downloadable-mods.json

```diff
@@ -0,0 +1,109 @@
+{
+  "mods": {
+    "kh1": [
+        { "repo": "SpaceBoiStreams/KH1-Sonic-Music-Pack" },
+        { "repo": "Napstio/KH1KairiMod-KairiKH1" },
+        { "repo": "Napstio/KH1KairiMod-KairiKH2" },
+        { "repo": "Napstio/KH1KairiMod-KairiKH3" },
```

Comment:
These three mods have Luas that do not work but work fine otherwise they are fine to stay just noting it. The model size will not be changed, names in menu wont be changed, and the other Lua seems to be a command menu change. but otherwise the model replacements and voice line stuff all work fine.